docs/CHANGELOG.asciidoc

```diff
@@ -26,6 +27,324 @@ This section summarizes the changes in each release.
+* Populate _id in tooltip {pull}41684[#41684]
+* Only show top hits checkbox if index has date fields {pull}43056[#43056]
+* Reposition tooltip when tooltip size changes {pull}43152[#43152]
+* Exclude index pattern from draw tools and shape filtering when layer ignores global filter {pull}43380[#43380]
+* Increase tooltip max width so create filter button not clipped {pull}43602[#43602]
+* Add i18n wrappers around missed text {pull}43635[#43635]
+* Only display fields contained in _source for Documents source term join left field options {pull}44386[#44386]
+* Fix side panel scrolling in safari {pull}44700[#44700]
+* Name space dynamic property style names to avoid collisions {pull}44676[#44676]
+* Unmount map embeddable component on destroy {pull}45183[#45183]
+* Set mapbox Popup maxWidth {pull}45253[#45253]
+Monitoring::
+* Ensure we pass down all the parameters for fetching logs {pull}43869[#43869]
+* Issue 25323: Fixed error catch and route handling v2 {pull}44800[#44800]
+* Respect xpack.monitoring.show_license_expiration {pull}45537[#45537]
+Operations::
+* Configurable global socket timeouts {pull}31603[#31603]
+* Only provide obsoleteIndexTemplatePattern to the default index migrator to avoid race conditions {pull}42016[#42016]
+Platform::
+* Modify I18nProvider so that it does not generate new React components {pull}43556[#43556]
+* Use pipes for communication with chrome to avoid networking snafus {pull}42097[#42097]
+* Fix rawChecksums in chromium build and rename for clarity {pull}44665[#44665]
+* Fix duplicate fetch in Visualize {pull}41204[#41204]
+* Ensure visualizations in percentage mode respect the advanced settings percent format {pull}39044[#39044]
+* Resolve filter index references when importing saved objects {pull}42974[#42974]
+Queries::
+* Saved query requires title {pull}45029[#45029]
+* Save query form validation on blur {pull}43726[#43726]
+Reporting::
+* Fix Relative Date format in CSV export {pull}40544[#40544]
+* Fix TypeError with pending job info {pull}43924[#43924]
+SIEM::
+* Toggle Column / Code Coverage and Cypress {pull}42766[#42766]
+* Show all popover items / Timeline flyout button rendering {pull}45745[#45745]
+* Upgrade elastic charts library - v12.0.2 {pull}45853[#45853]
+* Adds performance enhancements such by removing wasted renderers and adding incremental DOM rendering {pull}43157[#43157]
+* Fixes the Suricata row renderer not being activated {pull}44728[#44728]
+* Changes ML conditional links to use tabs, fixes a small bug with null filterQuery {pull}45218[#45218]
+* Fixes index substring incorrectly matching configured indices and failing to install ML job {pull}43409[#43409]
+* Fix timeline/kql search disparity {pull}42843[#42843]
+* Formats the bytes columns in timeline {pull}43147[#43147]
+* Capitalizes SIEM in {kib} Advanced Settings {pull}44886[#44886]
+* Removes expensive global CSS calculations of resize handle {pull}43898[#43898]
+Security::
```

Comment:
If we're going alphabetical, Security should go before SIEM.  Same is true in Enhancements section.